packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperfluid.sol

```diff
@@ -196,15 +199,16 @@ interface ISuperfluid {
     event AppRegistered(ISuperApp indexed app);
```

Comment:
we should mark registerApp as deprecated in the comments?

per https://docs.soliditylang.org/en/develop/natspec-format.html

maybe `custom: deprecated` is the closed we could use to mark it as deprecated

Reply:
also then we have to start to use registerAppWithKey in all our examples, and communicate this change to the devx folks.

Reply:
I know we wanted to deprecate it.
However working on it my thinking went like: `registerWithKey` won't make much sense anymore once (if) we switch to permissionless App registration. Thus it's probably better to just leave `registerApp` in place, but tell current App devs they can use the same register method on both testnet and mainnet (could be with key or by factory).

Reply:
but it makes sense "right now", because otherwise the app would need to use registerApp for testnet deployment, and registerWithKey on mainnet deployment. Since solidity doesn't have pre compiler, app would need to fiddle with this. And we don't really have examples for that.

Reply:
done, I also updated the docs and created https://github.com/superfluid-finance/protocol-monorepo/wiki/Super-App-White-listing-Guide